Chart docs: Add "Customizing Workers" page (#20331)
mistercrunch/Airflow: docs/helm-chart/customizing-workers.rst

 .. Licensed to the Apache Software Foundation (ASF) under one
    or more contributor license agreements.  See the NOTICE file
    distributed with this work for additional information
    regarding copyright ownership.  The ASF licenses this file
    to you under the Apache License, Version 2.0 (the
    "License"); you may not use this file except in compliance
    with the License.  You may obtain a copy of the License at

 ..   http://www.apache.org/licenses/LICENSE-2.0

 .. Unless required by applicable law or agreed to in writing,
    software distributed under the License is distributed on an
    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
    KIND, either express or implied.  See the License for the
    specific language governing permissions and limitations
    under the License.

Customizing Workers
===================

> Both ``CeleryExecutor`` and ``KubernetsExecutor`` workers can be highly customized with the :ref:`workers parameters <parameters:workers>`.

Author: ```suggestion
Both ``CeleryExecutor`` and ``KubernetesExecutor`` workers can be highly customized with the :ref:`workers parameters <parameters:workers>`.
```

For example, to set resources on workers:

.. code-block:: yaml

  workers:
    resources:
      requests:
        cpu: 1
      limits:
        cpu: 1

See :ref:`workers parameters <parameters:workers>` for a complete list.

Custom ``pod_template_file``
----------------------------

With ``KubernetesExecutor`` or ``CeleryKubernetesExecutor`` you can also provide a complete ``pod_template_file`` to configure Kubernetes workers.
This may be useful if you need different configuration between worker types for ``CeleryKubernetesExecutor``
or if you need to customize something not possible with :ref:`workers parameters <parameters:workers>` alone.

> As an example, let's say we want to set ``priorityClassName`` on our workers:

Reviewer: ```suggestion
As an example, let's say you want to set ``priorityClassName`` on your workers:
```
Very small nit here. This is written in first-person rather than second-person like the rest of the doc. Just trying to keep a consistent voice.

Author: Thanks!
